racingcar-java/src/main/java/racingcar/Cars.java

```diff
@@ -9,13 +9,24 @@ public class Cars {
 
     private final List<Car> cars;
 
-    public Cars(int size) {
+    public Cars(MoveStrategy moveStrategy, int size) {
         List<Car> temp = new ArrayList<>();
         IntStream.rangeClosed(1, size).
-                mapToObj(i -> new Car()).
+                mapToObj(i -> new Car(moveStrategy)).
                 forEach(temp::add);
         if(size <= 0)
-            temp.add(new Car());
+            temp.add(new Car(moveStrategy));
+        cars = Collections.unmodifiableList(temp);
+    }
+
+    public Cars(MoveStrategy moveStrategy, String string) {
+        List<Car> temp = new ArrayList<>();
+        List<Name> names = NameSpliter.nameSplit(string);
+        IntStream.range(0, names.size()).
+                mapToObj(i -> new Car(moveStrategy, names.get(i))).
+                forEach(temp::add);
```

Comment:
```suggestion
        IntStream.range(0, names.size())
                .mapToObj(i -> new Car(moveStrategy, names.get(i)))
                .forEach(temp::add);
```
사소하지만 개인적으로는 메소드 체이닝 라인변경을 해당 라인에 변경하는게 더 가독성 좋다라고 생각해요 :)

Reply:
컨벤션 배워갑니다. 감사합니다!!